react/components/EXPERIMENTAL_Modal/index.tsx

```diff
@@ -157,7 +157,7 @@ const ModalContent = forwardRef<HTMLDivElement, ContentProps>(
       // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
       <div
         className={classNames(
-          'vtex-modal__modal flex flex-column relative bg-white shadow-5 center mv9 br2',
+          `vtex-modal__modal flex flex-column relative bg-white shadow-5 center mv9 br2 ${styles.maxHeight100}`,
```

Comment:
It could be like this too ⛏️ 
```suggestion
          'vtex-modal__modal flex flex-column relative bg-white shadow-5 center mv9 br2',
          styles.maxHeight100,
```